### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by kevinlee12**

### CODE REVIEW

File: journal/static/journal/home/css/style.css [hunk 1]
```diff
@@ -13,50 +13,98 @@
...
-header {
-    position: absolute;
-    width: 95%;
-    padding-left: 10px;
-    line-height: 108px;
-    height: 108px;
-    background-color: rgba(0,0,0,0.1);
-    color: #fff;
-    z-index: 10;
+nav.navbar.navbar-default {
```
Comment (kevinlee12) [inline journal/static/journal/home/css/style.css]: space in between all of the elements

File: journal/static/journal/home/css/style.css [hunk 2]
```diff
@@ -69,13 +117,8 @@ h2 {
...
 }
 
-.container {
-    margin-top: 10px;
-    border: none;
-}
-
 .background {
-    background: url(../img/bg.jpg) no-repeat center center fixed;
+    background: url(../img/birdhouse.jpg) no-repeat center center fixed;
```
Comment (kevinlee12) [inline journal/static/journal/home/css/style.css]: No need to cite here, but did u keep the link of the source?

File: journal/templates/journal/home.html [hunk 3]
```diff
@@ -34,17 +53,22 @@
...
-            <section id="panel1">
-                <h2>More Stuff</h2>
-                <div id="categorization-rule">
-                    JZ, I'm going to leave it more or less like this, the rest is yours :)
-                </div>
-
+            <section id="about-us">
+                <h2>About Us</h2>
+            </section>
+            <section id="features">
```
Comment (kevinlee12) [inline journal/templates/journal/home.html]: Next week, you will need to add a line that divides the sections so that there is some sort of break.

**Comment by kevinlee12** [comment]
@janicez11, overall looks great, 1 spacing issue. Also, do remove the unneeded images (logos), we don't have a lot of disk real estate on our server.

File: journal/static/journal/home/css/style.css (hunk 1 above)
Comment (janicez11) [inline journal/static/journal/home/css/style.css]: I do have space in between each elements. which one are u talking about exactly?

Comment (kevinlee12) [inline journal/static/journal/home/css/style.css]: `nav.navbar.navbar-default ` should be `nav .navbar .navbar-default`. Please do check that the website doesn't break before pushing again, if a space breaks that, leave it.

Comment (janicez11) [inline journal/static/journal/home/css/style.css]: it breaks, so I am just going to leave it

**Comment by kevinlee12** [comment]
@janicez11 looks great merging!

**Pull request merged**